validate_yaml.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+"""
+Validates all yaml files by testing if entities and intents
+are unique and if all files mentioned are present
+"""
+
+import glob
+import os
+import sys
+
+import yaml
+from fire import Fire
+
+from validate_regex import validate_regex_in_yaml_file
+
+
+def print_or_throw_error(condition, success_message, failure_message, code=1):
```

Comment:
`Code` looks unused. Should it deleted or is this missing code?

Reply:
Used in check_key. Cleaning up some other references to outdated function `throw_error`.